.pre-commit-config.yaml

```diff
@@ -40,9 +40,10 @@ repos:
       - id: cmake-format
         name: CMake Format
         language: python
-        entry: bash -c "pip install cmake-format && python run-cmake-format.py --check"
-        entry: echo
-        files: ^(.*/CMakeLists.txt|.*.cmake)$
+        entry: cmake-format --in-place --autosort=false
+        files: CMakeLists\.txt$|^cpp/cmake_modules/
+        additional_dependencies:
+          - cmake_format==0.6.13
```

Comment:
Why do you specify `0.6.13`?
We're using 0.5.2 as I said at https://github.com/apache/arrow/pull/9045#discussion_r550335310 .

Anyway, we can't merge this until we fix the following issues:

  * https://github.com/apache/arrow/pull/9045#issuecomment-752475426
  * https://github.com/apache/arrow/pull/9045/checks?check_run_id=1629598956

Could you fix them?

Reply:
sorry, I glossed over that, fixed now

Reply:
> Anyway, we can't merge this until we fix the following issues:
> 
>     * [#9045 (comment)](https://github.com/apache/arrow/pull/9045#issuecomment-752475426)
> 
>     * https://github.com/apache/arrow/pull/9045/checks?check_run_id=1629598956
> 
> 
> Could you fix them?

Thanks for fixing the latter.
Could you also fix the former?